### ISSUE TITLE
Tweakers.net showing anti adblock + ads

### ISSUE THREAD

**Issue by BPower0036** [uBlockOrigin/uAssets#14224]
### Prerequisites

- [X] This is not a [support issue or a question](https://old.reddit.com/r/uBlockOrigin/)
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uAssets/issues) to avoid opening a duplicate issue
- [X] I am running the latest version of uBlock Origin

### I tried to reproduce the issue when...

- [X] uBO is the only extension
- [X] uBO with default lists/settings
- [X] using a new, unmodified browser profile

### URL(s) where the issue occurs

https://tweakers.net/videokaarten/vergelijken/

### Describe the issue

ads sometimes take 30 seconds to show up sometimes instantly sometimes shows nothing

Mostly in pricewatch it self
anti adblock popsup usually once a day then comes back after a day or so being annoying.

### Screenshot(s)

![afbeelding](https://user-images.githubusercontent.com/80090789/182531853-236f1dd0-460f-42da-a1de-75e053b6bb16.png)
![afbeelding](https://user-images.githubusercontent.com/80090789/182531874-5fda6ad7-8e98-4042-80ba-adeceb2e051b.png)

### uBlock Origin version

1.43.0

### Browser name and version

Firefox: 103

### Settings

```yaml
uBlock Origin: 1.43.0
Firefox: 103
filterset (summary):
network: 81003
cosmetic: 46740
scriptlet: 17507
html: 714
listset (total-discarded, last updated):
added:
NLD-0: 2442-29, 22h.54m
default:
user-filters: 2-0, never
ublock-filters: 32993-107, 3d.23h.9m
ublock-badware: 4215-0, 3d.23h.8m
ublock-privacy: 242-0, 3d.23h.7m
ublock-abuse: 76-0, 3d.23h.6m
ublock-unbreak: 1864-0, 3d.23h.5m
ublock-quick-fixes: 344-1, 19h.13m
easylist: 68901-589, 3d.22h.57m
easyprivacy: 27459-581, 3d.22h.57m
urlhaus-1: 5269-0, 19h.12m
plowe-0: 3638-3, 3d.22h.57m
filterset (user): [array of 2 redacted]
modifiedUserSettings: [none]
modifiedHiddenSettings: [none]
supportStats:
allReadyAfter: 254 ms
maxAssetCacheWait: 83 ms
```

### Notes

Posted via @Sylvanas-1977 in https://github.com/EasyDutch-uBO/EasyDutch/issues/27

**Comment by krystian3w** [comment]
Ad is loaded by inline script (so long script).

Maybe long work:

    tweakers.net###contentArea>div>div[id]:matches-css-after(content: /^Advert/)

I don't see anti-adblock for now.

**Comment by JobcenterTycoon** [comment]
The ads and the anti adblock popup are blocked on my end

**Comment by JobcenterTycoon** [comment]
Force update the filter lists it got fixed 3 hours ago https://github.com/EasyDutch-uBO/EasyDutch/commit/4c3a616015bad4e323ee75e3924bac96925c05e5

**Comment by krystian3w** [comment]
So this is invalid issue?

**Comment by Sylvanas-1977** [comment]
> So this is invalid issue?

It got fixed 3 hours ago i posted it 5 hours ago

**Comment by krystian3w** [comment]
Strange is hiding only one class and page abort inject ads and pop-up (bug in own JavaScript?).

**Comment by BPower0036** [comment]
> Force update the filter lists it got fixed 3 hours ago [EasyDutch-uBO/EasyDutch@4c3a616](https://github.com/EasyDutch-uBO/EasyDutch/commit/4c3a616015bad4e323ee75e3924bac96925c05e5)
> 
> https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches

It is not a fix. It only hides the element of the anti-adblock popup. In stead of hiding elements, we should do something with JavaScript, but I am not that good with that.

And @krystian3w I don't have multi-accounts on GitHub.

**Comment by krystian3w** [comment]
In Firefox possible use html filtering to remove long inline compressed / obfuscated code, but this need check all functions on page, maybe some need load neurralized `criteo`.
